cob/cob.c

```diff
@@ -842,11 +842,11 @@ int m0_cob_domain_destroy(struct m0_cob_domain *dom,
 	seg = m0_be_domain_seg(bedom, dom);
 	m0_be_0type_del_credit(bedom, &m0_be_cob0, cdid_str, &cred);
 	M0_BE_FREE_CREDIT_PTR(dom, seg, &cred);
-	m0_btree_destroy_credit(dom->cd_object_index,   &cred, 1);
-	m0_btree_destroy_credit(dom->cd_namespace,      &cred, 1);
-	m0_btree_destroy_credit(dom->cd_fileattr_basic, &cred, 1);
-	m0_btree_destroy_credit(dom->cd_fileattr_omg,   &cred, 1);
-	m0_btree_destroy_credit(dom->cd_fileattr_ea,    &cred, 1);
+	m0_btree_destroy_credit(dom->cd_object_index,   NULL, &cred, 1);
```

Comment:
Extra space before 'NULL' , please check for rest of the functions below

Reply:
It is because all the NULL's start in the same column.